components/AssetCard/AssetCard.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
So we don't actually need `AssetCard` here as `VaultCard` acts as the basis for all these different card types. If you notice inside the `VaultCard` folder there's an `adapters` folder, this is where `AssetCard` would go.

```tsx
/*
VaultCard
  adapters
    AssetCard
   */ 
// AssetCard.tsx
// just passs in the specific bits it needs to look and behave like an Asset
// feel free to add in props to the `VaultCard` render to add in the space for the extra components you may need (like `meta` and `action`)
...
return (
  <VaultCard
    title={title}
    meta={(
        <PerformanceHighlight
                amountSubtitle="LAST SALE PRICE"
                amount="6.2"
                performance="+100%"
           />    
    )}
    action={(
       <Button
            className="w-full text-center"
            size={Size.LARGE}
            kind={Kind.PRIMARY}
        >
           Invest in {title}
        </Button>
     </div>
    )}
  />
);
```